### DISCUSSION TITLE
screen resolution on boot

### DISCUSSION THREAD

**Discussion by eightwt** [helloSystem/hello#407] [General]
I'm new to BSD but know enough to be dangerous in Linux. I've been messing around with HelloSystem in VirtualBox and find it a lot of fun. But I haven't yet figured out how to set the display to use my full screen.

Right now I have xrandr -s 1366x768 in my .zshrc but that will only change the res if I open a terminal, and then the dock ends up in the middle of my screen until I open and close something else.

The screen display app in utilities doesn't seem to do anything, just puts up a box that's labeled default. The other question I have is how to add an app to the dock? 

I'm using hello-0.8.0_0H77-FreeBSD-14.0-amd64.iso.

**Comment by probonopd** [comment]
Hello @eightwt, please retry with [hello-0.8.0_0H218-FreeBSD-13.1-amd64.iso](https://github.com/helloSystem/ISO/releases/tag/experimental-13.1) or later.

**Reply by eightwt** [reply]
Love to, but I got a 404 error on this link :(

**Reply by probonopd** [reply]
This is the correct link: https://github.com/helloSystem/ISO/releases/tag/experimental-13.1

**Reply by eightwt** [reply]
Got it, thanks. I'll give it a try.

**Comment by PaulMcClernan** [comment]
Also have laptop(s) I've used with helloSystem where it defaults to not-mirroring display, using wrong video output, with no way to reach the display settings controls.
